<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomBusinessEventWidget</class>
 <widget class="QWidget" name="CustomBusinessEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="sourceLabelLayout" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="sourceLabel">
         <property name="text">
          <string>Source contains:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::client::desktop::HintButton" name="sourceLabelHint" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="deviceNameEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="captionLabelLayout" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="captionLabel">
         <property name="text">
          <string>Caption contains:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::client::desktop::HintButton" name="captionLabelHint" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="captionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="descriptionLabelLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description contains:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::client::desktop::HintButton" name="descriptionLabelHint" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::HintButton</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/common/widgets/hint_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
